src/UserBundle/Action/ApiIndex.php

```
@@ -13,17 +13,28 @@
-    /**
-     * @return array{}
-     */
-    #[Template('@SolidInvoiceUser/Api/index.html.twig')]
-    public function __invoke(Request $request): array
+    public function __construct(
+        private readonly FeatureGate $featureGate,
+        private readonly UpgradePromptProvider $upgradePromptProvider,
+    ) {
+    }
+
+    public function __invoke(Request $request): Response
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)